lib/manageiq/floe/workflow/runner/podman.rb

```diff
@@ -3,6 +3,32 @@ module Floe
     class Workflow
       class Runner
         class Podman < ManageIQ::Floe::Workflow::Runner
+          def run!(resource, env = {}, secrets = {})
+            raise ArgumentError, "Invalid resource" unless resource&.start_with?("docker://")
+
+            image = resource.gsub("docker://", "")
+
+            secret_guid = nil
+
+            if secrets && !secrets.empty?
+              require "securerandom"
+              secret_guid = SecureRandom.uuid
+              AwesomeSpawn.run!("podman", :params => ["secret", "create", secret_guid, "-"], :in_data => secrets.to_json)
```

Comment:
```suggestion
            params = ["secret", "create", secret_guid, "-"]
            logger.debug("Running podman: #{AwesomeSpawn.build_command_line("podman", params)}")
            AwesomeSpawn.run!("podman", :params => params, :in_data => secrets.to_json)
```

Now that I wrote that I feel like this is a common logging situation - I'm wondering if AwesomeSpawn itself should have a :log_execution parameter or something.